README.md

```diff
@@ -137,6 +137,13 @@ sudo pacman -S ccache gcc meson alsa-lib libpng sdl2 sdl2_net opusfile \
                fluidsynth
 ```
 
+``` shell
+# openSUSE
+sudo zypper install ccache gcc gcc-c++ meson alsa-devel libpng-devel \
+                    libSDL2-devel libSDL2_net-devel opusfile-devel \
+                    fluidsynth-devel
+```
```

Comment:
On the Pi, testing with openSUSE Tumbleweed, the repo does support:
- libpng16-devel
- libmt32emu-devel

Although I experienced garbled output using `libmt32emu-devel` testing the floating-point rendering output (which was fixed using the meson wrap), I still suggest we add MT-32 lib here.

Reply:
I tested in openSUSE Leap in a VM - `libpng-devel` is fine, it installs up-to-date version which is libpng16-devel. `libmt32emu-devel` is not available in openSUSE Leap…

Reply:
Ahh; glad you checked Leap! 
(resolving)

Reply:
On the other hand… it will be easier for Tumbleweed users and Leap 15.3 will be released quite soon… I added mt32emu to the list.